app/assets/stylesheets/application.scss

```diff
@@ -16,3 +16,8 @@
 
 @import 'bootstrap-sprockets';
 @import 'bootstrap';
+<<<<<<< HEAD:app/assets/stylesheets/application.scss
```

Comment:
You need to fix all conflicts 
- remove all redundant code
- remove all something like
```
<<<<<<< HEAD:app/assets/stylesheets/application.scss
=======
>>>>>>> chapter 7:app/assets/stylesheets/application.css
```
etc.